tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -220,6 +220,10 @@ def create_statistics_html(self, output_dir):
         Creates an statistics.html file which contains statistics information
         from the HTML generation process.
         """
+        def severity_order(severity):
+            severities = ['CRITICAL', 'HIGH', 'MEDIUM', 'LOW', 'UNSPECIFIED']
```

Comment:
Please add a docstring and highlight that the order of the list members influences the sorting.